### PR TITLE
Peer Review CSV spec test (+ upgrades)

### PR DESCRIPTION
**Pull request by WChrisK**

### CODE REVIEW

**Comment by WChrisK** [comment]
*NOTE*: This also contains the pull request from here: https://github.com/MarkUsProject/Markus/pull/2649

The only file that needs to be focused on is `peer_reviews_controller_spec.rb`

Overall I think it's a bit messy, but I figured I could get some comments on it and see what is the best way to clean this up (if possible)

File: spec/fixtures/files/peer_reviews/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+temp_peer_review.csv
```
Comment (david-yz-liu) [inline spec/fixtures/files/peer_reviews/.gitignore]: Why don't we make the test remove the file if it exists after it's done executing.

**Comment by david-yz-liu** [comment]
@WChrisK the test is a little on the long side, but I do like that it's checking that the upload/download formats are inverses of each other. I'll merge this in after you've addressed my comment.

File: spec/fixtures/files/peer_reviews/.gitignore (hunk 1 above)
Comment (david-yz-liu) [inline spec/fixtures/files/peer_reviews/.gitignore]: Yeah, I think it's safe to assume no one will use the name, but maybe prefix it with an underscore just to be safe.

I'm surprised it's automatically deleted, but I'll go with it.

**Pull request merged**